layers/+tools/geolocation/packages.el

```diff
@@ -59,8 +59,8 @@ to not have to set these variables manually when enabling this layer."
     (progn
       (spacemacs/declare-prefix "aw" "weather")
       (spacemacs/set-leader-keys
-        "aww" 'sunshine-forecast
-        "awW" 'sunshine-quick-forecast))
+        "agw" 'sunshine-forecast
```

Comment:
I am not sure about this binding, geolocation is still in tools so I would expect it to be under `spc a t g` rather than `spc a g`. Do you have any strong reasons to bind it on `spc a g w` instead?

Reply:
Bug.  I will fix.